### PR TITLE
3.1.0

### PR DESCRIPTION
**Pull request by vinh-hh**
## Summary

add raw_result
https://www.notion.so/holistics/RFC-2126-Correct-resolve-time-in-matches-f2136f69dd93426ca82b7428c45beefd

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ x ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as before)

## Breaking change
If this is a breaking change, state the changes.

## Checklist

Please check directly on the box once each of these are done

- [ x ] Update CHANGELOG.md
- [ ] Code Review

### CODE REVIEW

File: packages/date-parser/src/result.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ export default class Result {
       start: this.start,
       end: this.end,
       weekStartDay: this.weekStartDay,
+      rawResult: {
+        start: this.start ? luxonFromChronoStruct(this.start).toISO() : null,
+        end: this.end ? luxonFromChronoStruct(this.end).toISO() : null,
```
Comment (khanhhuy) [inline packages/date-parser/src/result.js:34]: Please add tests em

Comment (vinh-hh) [reply packages/date-parser/src/result.js:34]: added https://github.com/holistics/js/pull/33/commits/2b6ec84b4b26c1fc0fd5b008f91c8a99c7519f54

File: packages/date-parser/src/dateParserV3.test.js [hunk 2]
```diff
@@ -590,3 +590,45 @@ describe('default Chrono parsers should work well despite system timezone', () =
     expect(res.asTimestamp().end).toEqual('2019-12-26T05:33:00.000+09:00');
   });
 });
+
+describe('rawResult', () => {
+  const defaultOpts = { parserVersion: PARSER_VERSION_3, output: 'date', timezoneRegion: 'Asia/Seoul' };
+
+  it('output date return rawResult', () => {
+    const res = parse('2019-12-01 - 2019-12-02', new Date('2019-12-26T02:14:05Z'), { ...defaultOpts });
```
Comment (khanhhuy) [inline packages/date-parser/src/dateParserV3.test.js:598]: Should add a case with datetime input e.g. 2019-12-02 12:00:00 - 2012-12-03 23:59:59

**Comment by vinh-hh** [comment]
move to https://github.com/holistics/js/pull/34

**Pull request closed without merging**